src/ringbuf.h

```diff
@@ -14,7 +14,7 @@ typedef struct ringbuf ringbuf_t;
 typedef struct ringbuf_worker ringbuf_worker_t;
 
 int		ringbuf_setup(ringbuf_t *, unsigned, size_t);
-void		ringbuf_get_sizes(unsigned, size_t *, size_t *);
+void		ringbuf_get_sizes(const unsigned, size_t *, size_t *);
```

Comment:
I generally don't constify the non-pointer parameters. Normally, there is no benefit: it is of little value for the compiler and it does not add extra API guarantees.

Reply:
Actually, you do...line 124 of src/ringbuf.c declares that parameter const, and the compiler flagged it.

Reply:
Indeed. It was not supposed to be there. I suppose neither gcc nor clang picked it up because of the reasons stated above (`const` has no real effect in this case).